docs/root/api-v1/http_filters/fault_filter.rst

```diff
@@ -77,7 +77,10 @@ Delay
 
   {
     "type" : "...",
-    "fixed_delay_percent" : "...",
+    "fixed_delay_percent" : {
```

Comment:
We don't accept v1 changes anymore. Please revert all changes to v1 code, schema, docs, etc.